libraries/core-react/src/Divider/Divider.tsx

```diff
@@ -17,25 +17,25 @@ const StyledDivider = styled.hr<StyleProps>`
   height: ${(props) => props.dividerHeight};
 `
 
-type Props = {
-  /** Color variants
- @default medium */
+export type DividerProps = {
+  /** Color variants */
   color?: 'lighter' | 'light' | 'medium'
-  /** Vertical spacings
-   @default medium */
+  /** Vertical spacings */
   variant?: 'small' | 'medium'
+  /** @ignore */
   className?: string
 }
 
-export const Divider = forwardRef<HTMLHRElement, Props>((props, ref) => {
-  const { color = 'medium', variant = 'medium', className } = props
-  const styleProps = {
-    backgroundColor: tokens.color[color],
-    marginTop: tokens[variant].spacings.top,
-    marginBottom: tokens[variant].spacings.bottom,
-    dividerHeight: tokens.height,
-  }
-  return <StyledDivider {...styleProps} className={className} ref={ref} />
-})
+export const Divider = forwardRef<HTMLHRElement, DividerProps>(
+  ({ color = 'medium', variant = 'medium', className = '' }, ref) => {
```

Comment:
This is an anonymous function, maybe thats why you eslint complains when you remove manual definition of `displayName`?

Reply:
Could be, I’ll look into that

Reply:
It is indeed because it’s an arrow function. Normally javascript infers the function name from the variable that holds the arrow function, but in this case that does not work. So then we either have to refactor the components that use arrow functions, or just accept that we have to explicitly declare the `displayName` in these cases.

Reply:
Should be no problem to change it from arrow function to named function in most cases tho. Suggest we conform on on way of doing this and change our components using arrow functions to named functions then?

Reply:
Yes, I agree. I made an issue (#732)